geode-core/src/main/java/org/apache/geode/internal/HeapDataOutputStream.java

```diff
@@ -48,7 +49,7 @@
     org.apache.geode.internal.serialization.BufferDataOutputStream
     implements ObjToByteArraySerializer, ByteBufferWriter {
 
-  public HeapDataOutputStream(Version version) {
+  public HeapDataOutputStream(VersionOrdinal version) {
```

Comment:
same comment about retaining the use of Version in all serialization code.  We need assurance in place that we will never encounter an unknown ordinal during serialization/deserialization.

Reply:
fixed